docs/usage/introduction.rst

```diff
@@ -170,7 +170,7 @@ The following example opens a Jpg image and saves it as Png format :
 Three options groups are currently supported : quality, resolution and flatten.
 
 .. TIP::
-   Default values are 75 for Jpeg quality, 7 for Png compression level and 72 dpi for x/y-resolution.
+   Default values are 75 for Jpeg quality, 7 for Png compression level, 75 for webp quality and 72 dpi for x/y-resolution.
```

Comment:
Is it correct that WebP has a default quality of 75? Where did you find this value?

Reply:
The following lines are used to set the webp image quality:
```javascript
if (isset($options['webp_quality']) && in_array($format, array('webp'))) {
      $image->setImageCompressionQuality($options['webp_quality']);
}
```

and as per the [official documentation for `setImageCompressionQuality`](http://php.net/manual/en/gmagick.setcompressionquality.php), its `$quality`'s default value is 75